gamemode/config/hunger.lua

```diff
@@ -0,0 +1,12 @@
+GM.Config = GM.Config or  {}
+
+local Config = GM.Config
+
+Config.MAX_FOOD = 100
+Config.MAX_WATER = 100
+Config.HUNGER_DELAY = 5
+Config.FOOD_LOSS_AMOUNT = -1
+Config.WATER_LOSS_AMOUNT = -1
+Config.HUNGER_DAMAGE_AMOUNT = 10
```

Comment:
Not a fan of having every config value in the gamemode in a flat table, would prefer a structure like `Config.Hunger.MAX_FOOD`, what do other people think?

Reply:
I like the thought of there being separate configs within the Config table

Reply:
Is there no WATER_DAMAGE_AMOUNT ?

Reply:
hunger damage amount is shared for both of them right now